Makefile

```diff
@@ -108,7 +108,9 @@ get-operator-crds: var-require-all-OPERATOR_ORGANIZATION-OPERATOR_GIT_REPO-OPERA
 	$(MAKE) fix-changed
 
 gen-semaphore-yaml:
-	$(DOCKER_GO_BUILD) sh -c "go run ./hack/cmd/deps generate-semaphore-yamls"
+	$(DOCKER_GO_BUILD) sh -c "DEFAULT_BRANCH_OVERRIDE=$(DEFAULT_BRANCH_OVERRIDE) \
```

Comment:
have this also take in the `RELEASE_BRANCH_PREFIX` for the release branch regex

Reply:
Does make the tool a bit harder to run "by hand", but makes sense